src/pages/ahnurim/Main/Main.js

```diff
@@ -0,0 +1,156 @@
+/* eslint-disable */
+import React, { useEffect } from 'react';
+import { useState } from 'react';
+import '../Main/Main.scss';
+// import Comment from './component/Comment';
+import FeedData from './component/feed';
+// import Component from "./Component/Comment"
+function Main() {
+  const [comment, setComment] = useState('');
+  const [commentArray, setCommentArray] = useState([]);
+  const [commentList, setCommentList] = useState([]);
+  const [feedList, setFeedList] = useState([]);
+
+  useEffect(() => {
+    fetch('http://localhost:3000/data/commentData.json')
```

Comment:
```suggestion
    fetch('/data/commentData.json')
```
mock data 의 경로는 위와같이 localhost:3000을 생략할 수 있습니다